### PR TITLE
fix linechart

### PR DESCRIPTION
**Pull request by PietroChitto**
* improved linechart tooltip
* show message when there are 0 or 1 point
* tooltips don't exit the screen anymore
* added bar indicator
* add possibility to set min y

### CODE REVIEW

File: lib/custom_widgets/line_chart.dart [hunk 1]
```diff
@@ -10,21 +18,38 @@ class LineChartWidget extends StatefulWidget {
   final Color colorLine2Data;
 
   //Contains the number of days of the month
-  final double maxDays;
+  //final double maxDays;
 
+  // Used to decide the bottom labal
```
Comment (theperu) [inline lib/custom_widgets/line_chart.dart:23]: This might be a typo 🙂

File: lib/custom_widgets/line_chart.dart [hunk 2]
```diff
@@ -10,21 +18,38 @@ class LineChartWidget extends StatefulWidget {
   final Color colorLine2Data;
 
   //Contains the number of days of the month
-  final double maxDays;
+  //final double maxDays;
```
Comment (theperu) [inline lib/custom_widgets/line_chart.dart:21]: I think we can just remove this

**Review by theperu — APPROVED** [review]
Apart from two small things it looks good to me and we can merge it 😁

**Review by theperu — APPROVED** [review]

**Comment by theperu** [comment]
I think we can merge this since the failed check is not related to the linechart, @mikev-cw or @lucaantonelli do you agree?

**Review by lucaantonelli — APPROVED** [review]

**Pull request merged**